tasks/create-advisory/create-advisory.yaml

```diff
@@ -145,4 +150,6 @@ spec:
           exit 1
         fi
 
-        echo -n "$(echo ${results} | jq -r '.advisory_url // ""')" | tee $(results.advisory_url.path)
+        URL=$(echo ${results} | jq -r '.advisory_url // ""')
+        echo -n "$URL" | tee $(results.advisory_url.path)
+        echo '{"advisory":{"url":"'$URL'"}}' | tee $RESULTS_FILE
```

Comment:
Just an idea: Use `jq` to generate the json: `jq -n --arg url "$URL" '{"advisory": {"url": $url}}'`

The advantage is that it will be nicely formatted, so it's more readable in the log and/or the results file.

Reply:
Updated (with `-c`)

Reply:
The latter, after all the safe looping stuff multiline scares me

Reply:
I turn it to compact in the other task though so I will remove it here

Reply:
updated

Reply:
OK, cool. But I think it's not really even a problem in most cases:
```sh
$ MYJSON=$(jq -n --arg url "http://blabla.com" '{"advisory": {"url": $url}}')
$ echo $MYJSON
{ "advisory": { "url": "http://blabla.com" } }
```